xml/sle_update_upgrading.xml

```diff
@@ -621,583 +638,90 @@ tmpfs          506M     0  506M   0% /dev/shm
   </variablelist>
  </sect1>
  <sect1 xml:id="sec.update.sle12">
-  <title>Supported Methods for Upgrading &sle;</title>
-
-  <remark>toms 2014-02-11: See FATEs #315969 and #313189</remark>
-
+  <title>Online and Offline Upgrade</title>
   <para>
-   Upgrading from &sle;&nbsp;11 SP3 to &sle;&nbsp;12, &sle;&nbsp;11 SP3 to
-   &sle;&nbsp;12 SP1, or &sle;&nbsp;11 SP4 to &sle;&nbsp;12 SP1 is supported
-   using one of the following methods:
+   &suse; supports two different upgrade scenarios:
   </para>
 
+  <variablelist>
+   <varlistentry>
+    <term>Online</term>
+    <listitem>
+     <para>
+      Connected through &scc;, &smtool; (&smt;), or &susemgr;
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Offline</term>
+    <listitem>
+     <para>
+      Boot DVD, flash disk, ISO image, &ay;, <quote>plain RPM</quote> and
+      third-party tools
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   Upgrading from &sle; 11 SP4 to &sle; 12 SP3 is only supported via
+   an offline upgrade.
```

Comment:
I think there is a missing transition. Could you add an additional text to lead over the following itemizedlist? I think it would be enough to amend the text with: "... an offline upgrade using one of the following methods:"